nomt/src/store.rs

```diff
@@ -131,17 +133,14 @@ impl Transaction {
     }
 
     /// Write a page to storage in its entirety.
-    pub fn write_page<V: AsRef<[u8]>>(&mut self, page_id: PageId, value: V) {
-        let cf = self.shared.db.cf_handle(PAGES_CF).unwrap();
-        self.batch
-            .put_cf(&cf, page_id.length_dependent_encoding(), value);
+    pub fn write_page(&mut self, page_id: PageId, page: &Vec<u8>, page_diff: PageDiff) {
```

Comment:
`&[u8]` is better than `&Vec<u8>`!

Reply:
We can agree on that and that's a verbatim copy. We should make a clean-up sweep afterwards.